Mechanist Lair Expanded

mothheart opened the pull request:
Based on issue #134, not sure if the cleaning data is totally accurate as the nexus crashed as I was attempting to check for wild edits.

If I do find wild edits later down the line (this is one of the mods I routinely use), I'll update this commit.

Sharlikran committed:
Update Cleaning Info

Commit:
Added "Mechanist Lair Expanded" to the masterlist.

MacSplody commented on masterlist.yaml:
```diff
@@ -5403,3 +5403,59 @@ plugins:
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Mechanist Lair Expanded plugin' ]
+        condition: 'many("MechanistsLair_\w+\.esp")'
```
There's a missing newline.

```suggestion
        condition: 'many("MechanistsLair_\w+\.esp")'

```

MacSplody replied:
The shared meta data can be combined by using the regex.

```suggestion
  - name: 'MechanistsLair_\w+\.esp'
    url: [ 'https://www.nexusmods.com/fallout4/mods/11490' ]
    msg:
      - <<: *useOnlyOneX
        subs: [ 'Mechanist Lair Expanded plugin' ]
        condition: 'many("MechanistsLair_\w+\.esp")'
  - name: 'MechanistsLair_fix_only.esp'
    dirty:
      - <<: *quickClean
        crc: 0xEF7399E0
        util: '[FO4Edit v4.0.2j](https://www.nexusmods.com/fallout4/mods/2737)'
        itm: 13
  - name: 'MechanistsLair_V1.esp'
    dirty:
      - <<: *quickClean
        crc: 0x1E42B3E3
        util: '[FO4Edit v4.0.2j](https://www.nexusmods.com/fallout4/mods/2737)'
        itm: 11
  - name: 'MechanistsLair_V2.esp'
    dirty:
      - <<: *quickClean
        crc: 0x7DD23517
        util: '[FO4Edit v4.0.2j](https://www.nexusmods.com/fallout4/mods/2737)'
        itm: 11
  - name: 'MechanistsLair_V3.esp'
    dirty:
      - <<: *quickClean
        crc: 0x7DBD6DA2
        util: '[FO4Edit v4.0.2j](https://www.nexusmods.com/fallout4/mods/2737)'
        itm: 11
  - name: 'MechanistsLair_V4.esp'
    dirty:
      - <<: *quickClean
        crc: 0x2DFE73AE
        util: '[FO4Edit v4.0.2j](https://www.nexusmods.com/fallout4/mods/2737)'
        itm: 11
```

MacSplody committed:
Add missing newline at end of file

Commit:
Included shared metadata to prevent repetition.

Based on a comment by CptMcSplody (https://github.com/loot/fallout4/pull/148#discussion_r396165558)

Co-Authored-By: McSplody <1944639+CptMcSplody@users.noreply.github.com>

ghost commented:
I included your metadata change, it should also include the missing newline fix.

MacSplody commented:
@mothheart Thanks.
